modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -61,24 +63,37 @@ metadata:
 type: Opaque
 stringData:
  nmstate: |
```

Comment:
@jadhaj is this line supposed to be here? The `|`?

Reply:
Yes